specs/src/specs/namespace.md

```diff
@@ -58,25 +58,28 @@ The ID is encoded as a byte slice of length 28.
 ## Reserved Namespaces
 
 Celestia reserves certain namespaces with specific meanings.
-Celestia makes use of the reserved namespaces to properly organize and order transactions and blobs inside the [data square](./data_square_layout.md).
+Reserved namespaces are used to properly organize and order transactions and blobs inside the [data square](./data_square_layout.md).
 Applications MUST NOT use these reserved namespaces for their blob data.
 
-Below is a list of reserved namespaces, along with a brief description of each.
-In addition to the items listed in this table, it should be noted that namespaces with values less than `0x00000000000000000000000000000000000000000000000000000000FF` are exclusively reserved for use within the Celestia protocols.
+Reserved namespaces fall into two categories, _Primary Reserved Namespaces_ and _Secondary Reserved Namespaces_.
+- Primary Reserved Namespaces: Namespaces with values less than or equal to `0x00000000000000000000000000000000000000000000000000000000FF` are referred to as Primary Reserved Namespace and are exclusively reserved for use within the Celestia protocols.
+- Secondary Reserved Namespaces: Currently, there are two secondary reserved namespaces, namely,  `PARITY_SHARE_NAMESPACE` and `TAIL_PADDING_NAMESPACE` which fall out of the range of primary reserved namespaces.
```

Comment:
Do we ever want to give ourselves some padding so we can add secondary namespaces in the future? The same we do with primary?